packages/cyberstorm/src/components/Avatar/AvatarButton.tsx

```diff
@@ -27,11 +34,17 @@ export const AvatarButton = React.forwardRef<
       {...forwardedProps}
       ref={ref}
     >
-      <img
-        src={src}
-        className={classnames(styles.image, getSize(size))}
-        alt=""
-      />
+      {src ? (
```

Comment:
Could we just use Avatar component here to avoid duplication?

Reply:
We could. We'd just a have a double wrapper problem. I'll add a separate task for combining the components.